src/ballet/pack/fd_microblock.h

```diff
@@ -51,8 +51,16 @@ typedef struct fd_entry_batch_header fd_entry_batch_header_t;
 struct __attribute__((aligned(64))) fd_txn_p {
   uchar payload[FD_TPU_MTU];
   ulong payload_sz;
-  uint  requested_cus; /* Populated by pack. Bank does not read this. */
-  uint  executed_cus;  /* Populated by bank */
+  union {
+   struct {
+     uint non_execution_cus;
+     uint requested_execution_cus;
+   } pack_cu; /* Populated by pack. Bank reads these to populate the other struct of the union. */
+   struct {
+     uint rebated_cus; /* requested_execution_cus-real execution CUs. Pack reads this for CU rebating. */
+     uint total_executed_cus; /* non_execution_cus+real execution CUs. PoH reads this for block CU counting. */
```

Comment:
Maybe this should be called `actual_consumed_cus` or something like that. I'd rather not use `executed` since that seems similar to execution